Utilities/CodeGeneration/EdFi.Ods.CodeGen.Tests/Approval/4.0.0/DataStandard_400_ApprovalTests.Verify.Standard_Std_4.0.0_Artifacts_MsSql_Data_Security_0002-ResourceClaimMetadata_generated.approved.sql

```diff
@@ -867,7 +867,7 @@ BEGIN
 
     INSERT [dbo].[ResourceClaims] ([ResourceName], [ClaimName], [ParentResourceClaimId])
     VALUES (N'educationOrganizationNetworkAssociation', N'http://ed-fi.org/ods/identity/claims/educationOrganizationNetworkAssociation',
-@educationOrganizationsResourceClaimId);
+@relationshipBasedDataResourceClaimId);
```

Comment:
These changes look correct because the EducationOrganizationNetworkAssociation entity is not an EdOrg (but EducationOrganizationNetwork is).